backends/concrete-cpu/implementation/src/implementation/wop_simulation.rs

```diff
@@ -442,6 +448,29 @@ pub fn blind_rotate(
         security_level,
     );
 
+    let blind_rotate_variance = variance_blind_rotate(
+        lwe_dimension,
+        glwe_dimension,
+        polynomial_size,
+        pbs_log_base,
+        pbs_level,
+        ciphertext_modulus_log,
+        53,
+        variance_bsk,
+    );
+
+    let ppks_variance = estimate_packing_private_keyswitch(
+        0.,
+        variance_bsk,
+        fpks_log_base,
+        fpks_level,
+        glwe_dimension,
+        polynomial_size,
+        ciphertext_modulus_log,
+    );
+
+    let ggsw_variance = blind_rotate_variance + ppks_variance;
```

Comment:
in optimizer:
let variance_ggsw = pp_variance + br_variance / 2.;

Reply:
yes, the average casse is /2. In simulation, this is the worst case